draft-ietf-quic-qlog-main-schema.md

```diff
@@ -244,6 +246,57 @@ schema structured as:
     * `scenario` event type
     * `marker` event type
 
+# Abstract LogFile Class {#abstract-logfile}
+
+A Log file is intended to contain a collection of events that are in some way
+related. An abstract LogFile class containing fields common to all log files is
+defined. Each concrete log file format derives from this, extending it by
+defining semantics and any custom fields.
+
+~~~ cddl
+LogFile = {
+    file_schema: text
+    qlog_serialization_format: text
```

Comment:
I think we can probably just leave this as `serialization_format`, since we don't have the `qlog_` prefix anywhere else anymore :)

Reply:
agreed, done